x-pack/plugins/cases/public/components/all_cases/all_cases_generic.tsx

```diff
@@ -72,13 +73,14 @@ export const AllCasesGeneric = React.memo<AllCasesGenericProps>(
     caseDetailsNavigation,
     configureCasesNavigation,
     createCaseNavigation,
-    disabledStatuses,
+    hiddenStatuses,
     isSelectorView,
     onRowClick,
     updateCase,
     userCanCrud,
   }) => {
     const { actionLicense } = useGetActionLicense();
+    const initialFilterOptions = hiddenStatuses ? { status: CaseStatuses.open } : {};
```

Comment:
What will happen if `hiddenStatuses` contain the `open` status?

Reply:
Yeah things might get weird. Maybe if `hiddenStatuses` is not an empty array it should pick the first valid status for the `initialFilterOptions`?

Reply:
@cnasikas You are right. It wouldn't work for this scenario. Thanks for catching it!
I added this 2a9660722964c478266a143d6214ee4b7f5c0e92 commit that implements what @jonathan-buttner proposed. Could you take a look?

Reply:
And here is the uinit test: 0578154dc2f9f0685ab7b4879085dba5c77d791f

Reply:
Thanks!